apps/storybook/docs/f_recepten/imports.stories.mdx

```diff
@@ -0,0 +1,73 @@
+import {Meta} from '@storybook/addon-docs';
+
+<Meta title="Recepten/Imports"/>
+
+# Imports
+
+## Import Varianten
+
+Er zijn verschillende manieren om te [importeren](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).
```

Comment:
Eventueel link aanpassen naar https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#forms_of_import_declarations dan verwijst het meteen naar de sectie van verschillende manieren om te importeren.

Reply:
Het doel is net de algemene pagina te refereren niet het technische concrete detail.